libraries/botbuilder-ai/src/qnamaker-interfaces/qnamakerOptions.ts

```diff
@@ -64,4 +65,9 @@ export interface QnAMakerOptions {
      * Ranker types.
      */
     rankerType?: string;
+
+    /**
+     * A value indicating user's choice to for Strict Filters Join Operation. 
+     */
+    strictFiltersCompoundOperationType?: StrictFiltersCompoundOperationType;
```

Comment:
any tests covering this change?

Reply:
Yes, added two tests along with testdata to validate new field.

---
In reply to: [473518553](https://github.com/microsoft/botbuilder-js/pull/2718#discussion_r473518553) [](ancestors = 473518553)